crates/cext/src/transpiler/target.rs

```diff
@@ -468,13 +471,19 @@ impl TargetEntry {
             operation: StandardOperation::Gate(operation),
             params,
             map: Default::default(),
+            name: name_new,
         }
     }
 
-    pub fn new_fixed(operation: StandardGate, params: SmallVec<[Param; 3]>) -> Self {
+    pub fn new_fixed(
+        operation: StandardGate,
+        params: SmallVec<[Param; 3]>,
+        name_fixed: Option<String>,
```

Comment:
Same as the comment above, no need to add suffixes to the arguments here

```suggestion
        name: Option<String>,
```

Reply:
Changed.